ovirt_imageio/client/_options.py

```diff
@@ -69,7 +72,18 @@ def __new__(cls, name=None, args=(), config=False, required=False,
             help (str): Help message to show in the online help.
         """
         return tuple.__new__(
-            cls, (name, args, config, required, type, default, help))
+            cls, (name, args, config, required, action, type, default, help))
+
+    def make_kwargs(self):
+        option_kwargs = {
+            "dest": self.name,
+            "help": self.help,
+        }
+        if self.action:
+            option_kwargs["action"] = self.action
+        else:
+            option_kwargs["type"] = self.type
```

Comment:
Is this always correct or just special case of action="store_true" or action="store_false"?

Reply:
Try to avoid support for action= will probably be best.

Reply:
Add

   "default": None

So ensure that `store_false` will not set the default value.